applications/application-template.md

```diff
@@ -10,6 +10,12 @@
 
 > ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
 
+## Referral Program :moneybag: 
+
+You can find more information about the program [here](../README.md#moneybag-referral-program).
+- **Referrer:** Name of the Polkadot Ambassador or GitHub account of the Web3 Foundation grantee
+- **Payment Address:** BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) payment address. Please also specify the currency. (e.g. 0x8920... (DAI))
```

Comment:
I would put this near the end. Before or after `Additional Information`.

Reply:
Together with "Where did you hear about the Grants Program..."

Reply:
Move it before additional information